src/runtime_src/core/common/info_vmr.cpp

```diff
@@ -79,6 +79,9 @@ get_vmr_status(const xrt_core::device* device, vmr_status_type status)
     case vmr_status_type::boot_on_default:
       status_string = "Boot on default";
       break;
+    case vmr_status_type::has_fpt:
+      status_string = "Has fpt";
```

Comment:
would this "has fpt" and "boot on default" case sensitive and trim the while space before and after? this might be a separate PR if you think it is necessary.